hal/src/main/native/systemcore/FRCDriverStation.cpp

```diff
@@ -212,10 +213,10 @@ void JoystickDataCache::Update(const mrc::ControlData& data) {
   controlWord.dsAttached = data.ControlWord.DsConnected;
   controlWord.eStop = data.ControlWord.EStop;
 
-  auto mode = data.GetOpMode();
-  if (mode == "Test") {
+  opMode = data.GetOpMode();
+  if (opMode == "Test") {
     controlWord.test = true;
```

Comment:
Should this be the PR we remove the test and autonomous fields from the control word? And maybe even enabled too? Since its covered under purely the OpMode.

Reply:
Done.